src/server/mapping/mapping.js

```diff
@@ -0,0 +1,22 @@
+
+const fs = require("fs");
+const path = require("path");
+const jsonData = fs.readFileSync(path.resolve(__dirname, "HGNC_dataset.json"));
+const jsonContent = JSON.parse(jsonData);
+
+
+// map HGNC symbol to HGNC ID
+// After validation step, we can assume all symbols are valid
+// so that we can find a mapping for sure
+const mapping = function(symbol) {
+  let id = 0;
```

Comment:
So if the symbol isnt found you return 0?

I think it would be better to return null

Reply:
Since we call validate before mapping, all symbols will be found for sure.

Yes. I agree it'd be better to return null.